conduct.md

```diff
@@ -52,35 +52,35 @@ TODO: Write an introductory paragraph.
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is key to both successful and sustainable code. However it's important that all communication between students and team members follows the three pillars of Respect, Trust, and Humility. In general, students will *respect* their peers, *trust* their contributions and thoughts, and practice *humility* in all aspects of communication, in person, on GitHub, in the Discord, and otherwise.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should communicate in a professional and respectful manner.
+- Students should use words like "Please" and "Thank you".
+- Students will allow other members and teams speak freely.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not communicate in a way that offends or disrespects others.
+- Students should not interrupt others - we all have something to share.
+- Students should not show signs of passive aggressive behavior. Working as a team means keeping emotions out of work.
```

Comment:
I'm not sure about the last part, "Working as a team means keeping emotions out of work." It might promote tone policing. One of our textbooks, [Building Software Together](https://buildtogether.tech/important/), includes a short section on tone policing.

> **Tone policing** is a tactic used to shut down discussion by criticizing people for expressing emotion. It is often used against women, who are told to "calm down" when they show any sign of being angry at being excluded or not taken seriously. As [this article](https://thebias.com/2017/09/26/how-good-intent-undermines-diversity-and-inclusion/) by [Annalee Flower](https://flowerhorne.com/) says:

> People often reach for positive statements like "assume good intent" because they're worried about people being "shamed" over innocent mistakes. But society at large is already inclined to assume good intent in people with power and privilege–even when they're not demonstrating it. If you want to build a culture of "assuming good intent," start by assuming good intent in marginalized people.

> Assume that they already tried being nice. Assume that their feelings are valid. Assume that, after a lifetime of practice, they are responding to harmful behavior in the way that is safest for them. Prioritize that safety over the momentary discomfort people feel when they realize they've done something hurtful.

Thoughts?

Reply:
I didn't write this though I think you make a good point here. I think we should either remove that line or change it to specifically "negative emotions".